_authors/osci_2023/vishnu2411.markdown

```diff
@@ -0,0 +1,9 @@
+---
+name: T Vishnuvardhan Reddy
+short_name: Vishnu
+twitter: iamvishnuv7
+github: vishnu2411
+
+---
+
+**<Name>** <Something about you!>
```

Comment:
Please fill out this last line with a short introduction of yourself

Reply:
Added some introduction to the profile.